stable/ark/values.yaml

```diff
@@ -38,11 +38,13 @@ configuration:
     #  s3ForcePathStyle:
     #  s3Url:
     #  kmsKeyId:
+    #  resourceGroup:
+    #  storageAccount:
 
   backupSyncPeriod: 60m
-  gcSyncPeriod: 60m
-  scheduleSyncPeriod: 1m
-  resourcePriorities: []
+  metricsAddress: ":8085"
+  resticTimeout: 1m
```

Comment:
This seems very low. `ark server --help` shows a default of `1h`.

Reply:
changed